<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_displayname">
           <property name="text">
            <string>DisplayName</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_displayID">
           <property name="text">
            <string>Disply ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_username">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_displayUsername">
             <property name="text">
              <string>DisplayUsername</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_changeUser">
           <property name="text">
            <string>Change Username</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_password">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Displaypassword</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_changePassword">
           <property name="text">
            <string>Change Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_coursesTaken">
         <property name="text">
          <string>Courses Taken</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_coursesRequired">
         <property name="text">
          <string>Courses Required</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QListView" name="listView_coursesTaken"/>
       </item>
       <item>
        <widget class="QListView" name="listView_coursesRequired"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="text">
      <string>Return to menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
